capstone/src/main/java/com/google/univiz/api/representation/Deadline.java

```diff
@@ -7,4 +7,37 @@
  * for the purpose of graphing due dates.
  */
 @AutoValue
```

Comment:
```suggestion
@AutoValue
@GenerateTypeAdapter
```
and also add `import com.ryanharter.auto.value.gson.GenerateTypeAdapter;`